notification/src/config/init/ensure-adyen-webhook.js

```diff
@@ -0,0 +1,99 @@
+import fetch from 'node-fetch'
+import { serializeError } from 'serialize-error'
+import { getLogger } from '../../utils/logger.js'
+import config from '../config.js'
+
+const mainLogger = getLogger()
+
+async function ensureAdyenWebhook(adyenApiKey, webhookUrl, merchantId) {
```

Comment:
Can we write some tests to make sure this code doesn't fail?

Reply:
I could write the tests, but it would take time and the benefit is almost zero. This code is rarely changed and it just ensures one resource, so it's very straightforward to test manually.